inference-engine/src/gna_plugin/layers/gna_layer_info.hpp

```diff
@@ -122,6 +128,15 @@ class LayerInfo {
     bool isPower() const noexcept {
         return isOfType("power");
     }
+    float PowerExponent() const noexcept {
```

Comment:
please move that helper function somewhere else - layer info basically check that layer is of certain type() layer params can be extracted somewhere else, This methods cannot be noexcept - ideally they should check for dynamic_cast succeed - so i would not use layerInfo class for that stuff

Reply:
Thank you for pointing this out. I planned to fix it before pushing and I forgot.